_episodes/01-intro.md

```diff
@@ -220,4 +220,29 @@ weight in kilograms is now: 65.0
 > {: .solution}
 {: .challenge}
 
+> ## Seeing Data Types
+>
+> Python can show a variable's data type using type(variable). 
+> What are the data types of the following variables?
+>
+> ~~~
+> planet = 'Earth'
+> type(planet)
```

Comment:
Would you please move the `type(planet)` call here in line 230 and the other `type()` calls given in 232 and 234 to the Solution section, to encourage the learners to recall `type()` on their own.

Reply:
We would still like for you to move the `type()` call lines to the solution, so learners doing the exercise can remember that they need to use `type()` on their own. i.e.:
```
> ~~~
> planet = 'Earth'
> apples = 5
> distance = 10.5
> ~~~
> {: .language-python}
>
> > ## Solution
> > ~~~
> > type(planet)
> > type(apples)
> > type(distance)
> > ~~~
> > {: .language-python}
> >
> > ~~~
> > <class 'str'>
> > <class 'int'>
> > <class 'float'>
> > ~~~
> > {: .output}
```

Reply:
Hi Zack, thanks for the additional work--one further change would be appreciated: In your most recent commit https://github.com/swcarpentry/python-novice-inflammation/pull/766/commits/d097ea845347f9e1f9b24d7ab9ebf57002e99117 you removed the `type()` calls from the challenge question as requested, but we would still like to see the calls shown in the solution before showing the output. This should be possible using the syntax given in the above thread message. Thank you again!